ext/vscode/package.nls.json

```diff
@@ -18,6 +18,8 @@
     "azure-dev.commands.cli.deploy.title": "deploy (deploy the application's code to Azure)",
     "azure-dev.commands.cli.restore.title": "restore (restores application dependencies)",
 
+    "azure-dev.commands.cli.infra-create.title": "infra create (create Azure resources for the application)",
```

Comment:
Remind me: is "provision" different from "infra create"?

Reply:
If that is the case, that is also why initially I only added "provision" to the set of commands available for AZD. For two reasons: (a) list of commands displayed upon right-click on `azure.yaml` gesture is kind of long. In fact, we should probably discuss trimming it a bit, and (b) it is arguably confusing to have command aliases that have quite different names.

Reply:
I agree, it makes sense to me to standardize. `provision` sounds "better" IMO but `infra create` is paired with `infra delete`, AFAIK there's nothing like `unprovision`. Consequently it maybe would be best to dump `provision` in favor of _only_ `infra create`.

@savannahostrowski do you have any thoughts?

Reply:
Yeah - `provision` is just an alias for `infra create`. For now, I think I'd surface `provision` over `infra create` because that's the promoted path for users (e.g. it's in all our demos, promotional materials etc.)

Reply:
I agree that it doesn't seem consistent though so worth discussing with the team for a longer-term solution

Reply:
I've created #1563 to cover this. I'd like to get what we have merged and work on that separately.